libraries/AP_GPS/AP_GPS_NMEA.cpp

```diff
@@ -223,6 +223,13 @@ bool AP_GPS_NMEA::_have_new_message()
     if (_last_VTG_ms != 0) {
         _last_VTG_ms = 1;
     }
+    if (_last_GSA_ms != 0 &&
+        now - _last_GSA_ms > 150) {
```

Comment:
it looks like this makes GSA support compulsory? How can we be sure that all NMEA GPS modules support GSA?
I think we need to make this optional

Reply:
It doesn't make it mandatory, this only enters if the message was received before (it's the same as VTG).